### DISCUSSION TITLE
How to modify the attributes of an address object using SDK?

### DISCUSSION THREAD

**Discussion by devsrm** [PaloAltoNetworks/pan-os-python#401] [General]
Is there a function available to update the name, ip , description and tag of a specific address object in panorama using pan-os python SDK

**Comment by shinmog** [comment]
Yep:

https://pan-os-python.readthedocs.io/en/latest/module-objects.html#panos.objects.AddressObject

**Comment by OuticNZ** [comment]
@devsrm - Did you ever find the answer to this?  The link provided for me really doesn't answer your question.
